Indexer/Indexer.php

```diff
@@ -104,11 +104,12 @@ public function setIndexNamePrefix($indexNamePrefix)
         $this->indexNamePrefix = $indexNamePrefix;
     }
 
-    public function setApiSettings($application_id, $api_key)
+    public function setApiSettings($application_id, $api_key, $connection_timeout)
```

Comment:
$connection_timeout should have a default value given it didn't exist before and it is part of the public interface.